[2025-04-06 23:21:17,302] [INFO] [api_client.py:46] - 登录系统: http://localhost:8080/api/user/get_token/
[2025-04-06 23:21:17,486] [INFO] [api_client.py:56] - 登录成功，已获取token
[2025-04-06 23:21:17,487] [INFO] [api_client.py:95] - POST请求: http://localhost:8080/api/users/
[2025-04-06 23:21:17,499] [ERROR] [api_client.py:147] - 请求失败: 400 - {"detail":"用户名[testuser]已存在"}
